generators/app/templates/tests/moduleName-spec.js

```diff
@@ -3,7 +3,7 @@ const viewports = Terra.viewports('tiny', 'medium', 'large');
 
 describe('<%= moduleClassName %>', () => {
   describe('Default', () => {
-    beforeEach(() => browser.url('/#/<%= rootPath %>/<%= moduleName %>/default-<%= moduleName %>'));
+    before(() => browser.url('/#/raw/tests/<%= projectName %>/default-<%= moduleName %>'));
```

Comment:
/#/raw/tests/terra-projectName/projectName/default-moduleName

Reply:
This outputs as expected! the projectName value is the correct terra-prefix prepended to the moduleName provided